src/analysis/special_functions/pow.lean

```diff
@@ -907,6 +945,13 @@ begin
     rw [sqrt_eq_zero_of_nonpos h.le, rpow_def_of_neg h, this, cos_pi_div_two, mul_zero] }
 end
 
+lemma rpow_div_two_eq_sqrt {x : ℝ} (r : ℝ) (hx : 0 ≤ x) : x^(r/2) = x.sqrt^r :=
```

Comment:
```suggestion
lemma rpow_div_two_eq_rpow_sqrt {x : ℝ} (r : ℝ) (hx : 0 ≤ x) : x^(r / 2) = (sqrt x) ^ r :=
```
looks more readable to me (in usual math formula, we don't use dot notation!) (unchecked, might failed if real is not open...)